Sources/Models/PlaygroundGenerator.swift

```diff
@@ -57,7 +57,104 @@ struct PlaygroundGenerator {
         self.latestVersion = tag.description
     }
 
-    func generate() {
-        // TODO: Implement here
+    func generate() throws -> URL {
+        let manifest = """
+            // swift-tools-version: 5.10
+
+            // WARNING:
+            // This file is automatically generated.
+            // Do not edit it by hand because the contents will be replaced.
+
+            import PackageDescription
+            import AppleProductTypes
+
+            let package = Package(
+                name: "My App",
+                platforms: [
+                    .iOS("17.5")
+                ],
+                products: [
+                    .iOSApplication(
+                        name: "My App",
+                        targets: ["AppModule"],
+                        displayVersion: "1.0",
+                        bundleVersion: "1",
+                        appIcon: .placeholder(icon: .box),
+                        accentColor: .presetColor(.blue),
+                        supportedDeviceFamilies: [
+                            .pad,
+                            .phone
+                        ],
+                        supportedInterfaceOrientations: [
+                            .portrait,
+                            .landscapeRight,
+                            .landscapeLeft,
+                            .portraitUpsideDown(.when(deviceFamilies: [.pad]))
+                        ]
+                    )
+                ],
+                dependencies: [
+                    .package(url: "\(self.packageURL)", from: "\(self.latestVersion)")
+                ],
+                targets: [
+                    .executableTarget(
+                        name: "AppModule",
+                        dependencies: [
+                        ],
+                        path: ".",
+                        swiftSettings: [
+                            .enableUpcomingFeature("BareSlashRegexLiterals")
+                        ]
+                    )
+                ]
+            )
+            """
+        let myApp = """
+            import SwiftUI
+
+            @main
+            struct MyApp: App {
+                var body: some Scene {
+                    WindowGroup {
+                        ContentView()
+                    }
+                }
+            }
+            """
+        let contentView = """
+            import SwiftUI
+
+            struct ContentView: View {
+                var body: some View {
+                    Text("Hello, world!")
+                }
+            }
+            """
+        let fm = FileManager.default
+        let tmp = URL.temporaryDirectory
+        let packageURL = tmp.appending(path: "My App.swiftpm")
+        if fm.fileExists(atPath: packageURL.path()) {
```

Comment:
The default argument of `path()` is `percentEncoded: true`, so this is not correct.

I have to change this line like:

```suggestion
        if fm.fileExists(atPath: packageURL.path(percentEncoded: false)) {
```

Reply:
Fixed. https://github.com/kkebo/try-in-a-playground.swiftpm/compare/f3b02bd3dc4ce3d6012004d90ee2960de214c31d..231ea616c5a9a21a69d48b8df40e99033e40fd51